objects/ogamemanager/Step_0.gml

```diff
@@ -116,7 +116,6 @@ switch (global.gameMode)
 			else if(gameHaltTimer == 220)
 			{
 				room_goto(rCharacterSelectScreen);
-				audio_stop_sound(testBGM);

```

Comment:
The other cases of going back to the character select screen have the line audio_group_stop_all(audiogroup_mus). Include it here too for consistency. Don't forget your semicolons.